pkgs/development/web/nodejs/v23.nix

```diff
@@ -53,5 +53,15 @@ buildNodejs {
       hash = "sha256-gmIyiSyNzC3pClL1SM2YicckWM+/2tsbV1xv2S3d5G0=";
       revert = true;
     })
+    # Fix for https://github.com/NixOS/nixpkgs/issues/355919
+    # FIXME: remove after a minor point release
+    (fetchpatch2 {
+      url = "https://github.com/nodejs/node/commit/a094a8166cd772f89e92b5deef168e5e599fa815.patch?full_index=1";
+      hash = "sha256-5FZfozYWRa1ZI/f+e+xpdn974Jg2DbiHbua13XUQP5E=";
+    })
+    (fetchpatch2 {
+      url = "https://github.com/nodejs/node/pull/55863/commits/f60cc5871b3950a078356dbd30cb16ba64890916.patch?full_index=1";
```

Comment:
We should wait for this to land upstream

Reply:
We as a packaging distro, have the ability and right to cherry-pick patches or hold back updates if needed. 

I'd rather not leave this broken in master and already branched-off release-24.11 for long. 

This is breaking dependent packages and should be done on part of ZHF.

Reply:
We do prefer in‐tree vendoring for unmerged patches, though.

Anyway, @aduh95 is an upstream Node developer and sent the PR you’re vendoring here, so I expect that it shouldn’t take too long to land, and it doesn’t seem very high‐priority as basically nothing uses this version.

Reply:
It seems like Zed is almost the only thing using this. Do they really require a non‐LTS Node? It seems better to pin it to Node 22 if possible.

Reply:
Sorry for trying to rush this. We are already near the end of release cycle and did not want to try to ship a broken zed-editor and nodejs as upstream release could be delayed.

But anyways, looks like Vicuna could be [delayed](https://github.com/NixOS/nixpkgs/pull/356133).

As for the zed-editor issue, perhaps CC @GaetanLepage

Reply:
I am not sure the `.netrc` thing is quite release‐delaying either. That PR cannot go to `staging-next` because there is no `staging-next`; the last cycle before release has already finished.

Reply:
> It seems like Zed is almost the only thing using this. Do they really require a non‐LTS Node? It seems better to pin it to Node 22 if possible.

No, we don't. I also originally requested in the PR introducing nodejs to Zed to use an LTS version of Node instead.

Reply:
Thanks. Then I think we should fix Zed and not worry too much about this.

Reply:
https://github.com/NixOS/nixpkgs/pull/356459

Reply:
The patch to zed has been merged and backported to 24.11.